lib/client/connection.js

```diff
@@ -255,6 +253,9 @@ Connection.prototype.handleMessage = function(message) {
       return;
 
     default:
+      if (this._connectionPresence.isPresenceMessage(message)) {
```

Comment:
I don't like leaning on `default` for this - why can't we give it its own message type?